chpl/chpl-service/pom.xml

```diff
@@ -418,8 +418,15 @@
             <artifactId>jwks-rsa</artifactId>
             <version>0.4.0</version>
         </dependency>
-
-        <!--  needed for unit tests -->
+		
```

Comment:
This diff is hard to read because of the checkstyle alerts here. Can you address those?